Add Pagination cookbook recipe

Tobbe opened the pull request.

Tobbe commented:
![image](https://user-images.githubusercontent.com/30793/94778130-ec6d6e00-03c4-11eb-9fd0-97cbcdf68ec2.png)

This image is used in the docs, so putting it here so that I can link to it

guledali commented:
@Tobbe Syntax highlighting is missing from this codeblock, I also think it's duplicate can probably be removed

<img width="879" alt="Screenshot 2020-10-01 at 09 21 49" src="https://user-images.githubusercontent.com/20647282/94779832-d319f100-03c7-11eb-809a-fda6ba7034a5.png">

Tobbe commented:
@guledali Yeah, it's missing highlighting because redwoodjs.com uses highlight.js which unfortunately doesn't support graphql highlighting 🙁 (github's markdown parser/viewer does, and so does the one I use on my blog). But I've changed it to javascript (for now) and removed the duplicate. Thanks for letting me know! 👍

cannikin commented:
There's a plugin for highlight.js that adds graphql support: https://www.npmjs.com/package/highlightjs-graphql

Tobbe commented:
Thanks for that link @cannikin. I made a separate PR to add support for graphql highlighting: https://github.com/redwoodjs/redwoodjs.com/pull/389

cannikin commented:
Look great, thanks!
